README.md

```diff
@@ -137,6 +138,7 @@ Flags:
 | Argument          | Description                                                                                               | Default Value |
 |-------------------|-----------------------------------------------------------------------------------------------------------|:-------------:|
 | **--chown**       | chown the output directory to match the specified UID:GID                                                 |       ❌      |
+| --progress        | Show progress in the given format, supported: plain,term,debug. If empty it is auto-detected              |       ❌      |
```

Comment:
Future: Should we have an `auto` value which is the explicit default?
(not very important)

Reply:
I updated this and also included the suggesion from Ondrej to rename "plain" to "verbose"